src/translib/transformer/xfmr_intf.go

```diff
@@ -2224,14 +2224,12 @@ var YangToDb_intf_eth_port_config_xfmr SubTreeXfmrYangToDb = func(inParams XfmrP
                 /* Check if given iface already part of a PortChannel */
                 err = validateIntfAssociatedWithPortChannel(inParams.d, &ifName)
                 if err != nil {
-                    errStr := "Interface already part of a PortChannel"
-                    return nil, tlerr.InvalidArgsError{Format: errStr}
+                    return nil, tlerr.InvalidArgsError{Format: err.Error()}
```

Comment:
can't we just do return nil, err. Because, we already pass tlerr instance as part of the function.

Reply:
done